gensim/models/wrappers/ldamallet.py

```diff
@@ -87,8 +88,15 @@ def __init__(self, mallet_path, corpus=None, num_topics=100, alpha=50, id2word=N
             Collection of texts in BoW format.
         num_topics : int, optional
             Number of topics.
-        alpha : int, optional
-            Alpha parameter of LDA.
+        alpha : float, optional
+            Alpha parameter of LDA (scalar for a symmetric prior over document-topic probability).
+            Default: 5.0 / num_topics
+        use_symmetric_alpha : bool, optional
+            Only optimize the concentration parameter of the prior over document-topic distributions.
+            (This may reduce the number of very small, poorly estimated topics,
+            but may disperse common words over several topics.)
+        eta : float, optional
+            Eta parameter of LDA (scalar for a symmetric prior over topic-word probability).
```

Comment:
Where does the `0.01` default come from?

Reply:
It is the Mallet default behaviour (see [here](https://github.com/mimno/Mallet/blob/024a64ff9caf3f54586c5f70399da001f208305b/src/cc/mallet/topics/ParallelTopicModel.java#L93)):
`public static final double DEFAULT_BETA = 0.01;`

Reply:
I see. And this one is not a "sum", unlike alpha?

Reply:
You are right, as mentioned here #2903
Beta parameter: smoothing parameter for each topic-word. beta_w = [this value]

Reply:
Alright. So please change `alpha` to be stored internally as non-sum too (convert to sum only when needed) and we can merge. Thanks.